civictechprojects/models.py

```diff
@@ -94,7 +96,7 @@ def hydrate_to_tile_json(self):
             'project_id': self.id,
             'project_name': self.project_name,
             'project_creator': self.project_creator.id,
-            'project_description': self.project_description,
+            'project_description': self.project_short_description if self.project_short_description else self.project_description,
```

Comment:
I like this. I was envisioning bringing them both out to the React frontend and picking there but this is nicer. 👍